test/helpers/quictestlib.c

```diff
@@ -399,19 +414,27 @@ int qtest_wait_for_timeout(SSL *s, QUIC_TSERVER *qtserv)
      */
     if (!SSL_get_event_timeout(s, &tv, &cinf))
         return 0;
-    if (using_fake_time)
+    if (using_fake_time) {
```

Comment:
We should probably move all this locking into `qtest_get_time` and `qtest_set_time` to go along with the existing `qtest_add_time`.

Reply:
updated to create qtest_reset_time and qtest_get_time, to isolate the locking